planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/utils/utils.cpp

```diff
@@ -287,6 +287,36 @@ bool isPathInLanelets(
   return true;
 }
 
+bool pathFootprintExceedsTargetLaneBound(
+  const CommonDataPtr & common_data_ptr, const PathWithLaneId & path, const VehicleInfo & ego_info,
+  const double margin)
+{
+  if (common_data_ptr->direction == Direction::NONE || path.points.empty()) {
+    return false;
+  }
+
+  const auto & target_lanes = common_data_ptr->lanes_ptr->target;
+  const bool is_left = common_data_ptr->direction == Direction::LEFT;
+
+  const auto combined_target_lane = lanelet::utils::combineLaneletsShape(target_lanes);
+
+  auto it = path.points.rbegin();
+  for (; it < path.points.rend(); ++it) {
+    const auto & pose = it->point.pose;
+    const auto & front_vertex = getEgoFrontVertex(pose, ego_info, is_left);
+
+    const double sign = is_left ? -1.0 : 1.0;
```

Comment:
```suggestion
    const auto sign = is_left ? -1.0 : 1.0;
```
Suggest using auto most of the time for consistency.